src/core/Slider/Slider.test.tsx

```diff
@@ -0,0 +1,934 @@
+  expect(
+    (container.querySelector('.iui-slider-max') as HTMLSpanElement).textContent,
+  ).toBe('100');
+});
+it('should render specified min max labels', () => {
+  const { container } = render(
+    <Slider values={defaultSingleValue} min={5} max={55} />,
+  );
+  assertBaseElement(container);
+  expect(
+    container.querySelector('.iui-slider-component-container'),
+  ).toBeTruthy();
+  expect(
+    (container.querySelector('.iui-slider-min') as HTMLSpanElement).textContent,
+  ).toBe('5');
+  expect(
+    (container.querySelector('.iui-slider-max') as HTMLSpanElement).textContent,
+  ).toBe('55');
+});
+it('should render provided min max labels', () => {
+  const { container } = render(
+    <Slider values={defaultSingleValue} maxLabel='big' minLabel='small' />,
+  );
+  assertBaseElement(container);
+  expect(
+    container.querySelector('.iui-slider-component-container'),
+  ).toBeTruthy();
+  expect(
+    (container.querySelector('.iui-slider-min') as HTMLSpanElement).textContent,
+  ).toBe('small');
+  expect(
+    (container.querySelector('.iui-slider-max') as HTMLSpanElement).textContent,
+  ).toBe('big');
+});
+it('should render provided min max label nodes', () => {
+  const { container } = render(
+    <Slider
+      values={defaultSingleValue}
+      maxLabel={<span className='span-max'>big</span>}
+      minLabel={<span className='span-min'>small</span>}
+    />,
+  );
+  assertBaseElement(container);
+  expect(
+    container.querySelector('.iui-slider-component-container'),
+  ).toBeTruthy();
+  expect(
+    (container.querySelector('.span-min') as HTMLSpanElement).textContent,
+  ).toBe('small');
+  expect(
+    (container.querySelector('.span-max') as HTMLSpanElement).textContent,
+  ).toBe('big');
+});
+it('should render provided min max label nodes', () => {
+  const { container } = render(
+    <Slider
+      values={defaultSingleValue}
+      maxLabel={<span className='span-max'>big</span>}
+      minLabel={<span className='span-min'>small</span>}
+    />,
+  );
+  assertBaseElement(container);
+  expect(
+    container.querySelector('.iui-slider-component-container'),
+  ).toBeTruthy();
+  expect(
+    (container.querySelector('.span-min') as HTMLSpanElement).textContent,
+  ).toBe('small');
+  expect(
+    (container.querySelector('.span-max') as HTMLSpanElement).textContent,
+  ).toBe('big');
+});
+it('should set focus', () => {
+  let element: HTMLDivElement | null = null;
+  const onRef = (ref: HTMLDivElement) => {
+    element = ref;
+  };
+  const { container } = render(
+    <Slider ref={onRef} values={defaultSingleValue} setFocus />,
+  );
+  assertBaseElement(container);
+  expect(element).toBeTruthy();
+  const thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+  expect(thumb).toBeTruthy();
+  expect(document.activeElement).toEqual(thumb);
+});
+it('should show tooltip when focused', () => {
+  let element: HTMLDivElement | null = null;
+  const onRef = (ref: HTMLDivElement) => {
```

Comment:
Any reason checking ref here ?

Reply:
removed all refs but one.... sloppy copy/pasting to start new test.